haas/ext/obm/ipmi.py

```diff
@@ -87,6 +87,15 @@ def power_off(self):
             raise OBMError('Could not power off node %s', self.label)
 
     @no_dry_run
+    def set_bootdev(self, dev):
+        valid_bootdevices = ['disk', 'pxe', 'none']
+        if dev not in valid_bootdevices:
+            raise OBMError('Invald boot device')
```

Comment:
How about I throw a `BadArgumentError` (400) which we already have instead of making a new one?